craft/pio/src/pio.scala

```diff
@@ -36,9 +36,13 @@ class Lpio(c: pioParams)(implicit p: Parameters) extends LpioBase(c)(p)
 {
 
 // User code here
+  override def extraResources(resources: ResourceBindings) =
+      Map("pio-width" -> Seq(ResourceInt(pioWidth)),
+          "compatible" -> Seq(ResourceString("sifive,blenderific111")))
```

Comment:
You shouldn't set the `compatible` property here, since that's what the second argument to `SimpleDevice()` is supposed to be.

@albertchen-sifive, we might want to add a runtime check to the base classes that prevents setting the `compatible` via this map.

Reply:
fixed

Reply:
> @albertchen-sifive, we might want to add a runtime check to the base classes that prevents setting the compatible via this map.

I think we do want people to be able to override this though? does adding the `"compatible"` mapping override any existing mapping or append to it?

Reply:
It overwrites.

Reply:
You make a good point about allowing people to override it. We'll just need to make sure our documentation in this is clear about the default value that it gets.